testbed/addons/kit/construct.ts

```diff
@@ -0,0 +1,97 @@
+import cdk = require('@aws-cdk/core')
+import eks = require('@aws-cdk/aws-eks')
+import iam = require('@aws-cdk/aws-iam')
+
+export interface KitProps {
+    cluster: eks.Cluster
+}
+
+export class Kit extends cdk.Construct {
+    constructor(scope: cdk.Construct, id: string, props: KitProps) {
+        super(scope, id)
+        const namespace = "kit"
+        const ns = props.cluster.addManifest('kit-namespace', {
+            apiVersion: 'v1',
+            kind: 'Namespace',
+            metadata: {
+                name: namespace
+            }
+        })
+
+        // Controller Role
+        const sa = props.cluster.addServiceAccount('kit-controller-sa', {
+            name: "kit-controller",
+            namespace: namespace
+        })
+        sa.node.addDependency(ns)
+        sa.role.attachInlinePolicy(new iam.Policy(this, 'kit-controller-policy', {
```

Comment:
Let's also make the policy name cluster specific as it will conflict with existing policies for other clusters running KIT.